install_mpremote.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Install mrequests to a MicroPython board using mpremote
+
+MODULES=('defaultdict.py' 'mrequests.py' 'urlencode.py' 'urlparseqs.py' 'urlunquote.py')
+DESTDIR=":/lib"
```

Comment:
Please update `install.sh` (the rshell variant) to use this dest dir as well (i.e. `/pyboard/lib`).

Reply:
I don't have any STM32 boards handy.
Do they still map `/pyboard`?
I thought the FS had been standardised to just have `/lib` in root.
I have an original PyBoard somewhere, can run some tests, but if you have one running let me know if this is the case.
First come > first served :)

Reply:
What I meant is that rshell uses the `/pyboard` prefix as an equivalent to `:`.

So, in `install.sh` we should change `"${DESTDIR:-/pyboard}"` to `"${DESTDIR:-/pyboard/lib}"`.